member-service/src/main/java/com/hedvig/memberservice/web/CashbackController.java

```
@@ -61,7 +62,7 @@ public CashbackController(
       return ResponseEntity.ok(cashbackOptions);
     }
 
-    return ResponseEntity.ok(cashbackService.getOptions());
+    return ResponseEntity.ok(cashbackService.getOptions(PickedLocale.sv_SE));
```

Should we default to a dummy option that only have empty data?